packages/clay-navigation/src/BreadcrumbEllipsis.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import React, {useState} from 'react';
+import {IBreadcrumbItem} from './BreadcrumbItem';
+
+export interface IBreadcrumbEllipsisProps {
+	/**
+	 * Property to define BreadcrumbEllipsis's items.
+	 */
+	items?: Array<IBreadcrumbItem>;
+
+	/**
+	 * Path to the location of the spritemap resource.
+	 */
+	spritemap: string;
```

Comment:
This should be optional since spritemap can come via context

Reply:
Makes sense. I'm currently updating all components that receive `spritemap` to be optional and update docs.